examples/basic/package.json

```diff
@@ -5,7 +5,7 @@
     "@babel/runtime": "^7.1.2",
     "babel": "^6.23.0",
     "react": "16.8.3",
-    "react-native": "0.59.1"
+    "react-native": "^0.59.1"
```

Comment:
@rfischer the same changes needs to be done for `/examples/mlkit`. I am working on it, let me know if you want to add it to this pull request.

Reply:
@fabriziobertoglio1987 I never tried the mlkit example so far. But if the same fix is also needed there, then you can add it to this pull request.

Reply:
https://github.com/fabriziobertoglio1987/react-native-camera/commit/52ec1dfb830d753d5225c4c315c20e9e7bb93a60 otherwise I publish the pull request. You need to register in firebase and download the `GoogleServices-Info.plist` file, but don't mind this .. I will resolve this comment as soon as I publish the pull request.. Just a reminder that it has to be done. Thanks a lot 
![image](https://user-images.githubusercontent.com/24992535/78780915-a4ced200-799f-11ea-80a3-02fb9db1237f.png)